Jenkinsfile

```diff
@@ -517,12 +520,26 @@ def doDockerInside(script, target, postStep = null) {
   }
 }
 
+def runEmulator() {
+  sh """yes '\n' | avdmanager create avd -n CIRJSEmulator -k 'system-images;android-29;default;x86' --force"""
```

Comment:
Wouldn't that be to ensure that this command never fails?  It will always evaluate to success, no matter whether `avdmanager ...` fails.

Reply:
that's a directly copy/pasted from what Java team uses.

Reply:
It just pipes in "yes /n" as the input to a cli-question, right?

Reply:
`yes '\n' |` will pipe a continuous stream of new line characters into STDIN of the `avdmanager` process, presumably to ensure that all prompts are agreed to or something. The reason I'm asking is that I think there's other standard runtime parameters that one can use to achieve this.

I don't think "because javas does it" qualifies as sufficient reason to add this in. This is working around some unknown issue that we as maintainers of the Jenkinsfile need to be aware of. If we cannot find another way to achieve the goal, the least we can do to our fellow developer is to add a note on the origin of this.

I suggest removing it and investigate any issues that might or might not arise from it and document the workaround.

Reply:
yes

Reply:
I don't understand the issue here.